api/views.py

```diff
@@ -30,14 +32,17 @@ def post(self, request: request.Request):
 class UserLoginView(views.APIView):
     permission_classes = (permissions.AllowAny,)
 
-    def post(self, request: request.Request):
+    def post(
+        self,
+        request: request.Request,
+    ):
         username = request.data["username"]
         password = request.data["password"]
 
         user = authenticate(request=request, username=username, password=password)
         if user is not None:
             login(request=request, user=user)
-            return response.Response(status=status.HTTP_200_OK)
+            return HttpResponseRedirect(redirect_to="/dashboard/")
```

Comment:
I was fiddling around with the login bug and found that this is where the problem is.
The redirect is a good idea, but I tested it and I can't get it to redirect. It only stays on the Token page after login. have you had different results?

Reply:
Yeah I've been able to redirect